sdk/monitor/Azure.Monitor.OpenTelemetry/src/AzureMonitorOpenTelemetryImplementations.cs

```diff
@@ -37,6 +38,26 @@ internal static IServiceCollection AddAzureMonitorOpenTelemetryWithAction(IServi
                             .AddHttpClientInstrumentation()
                             .AddAzureMonitorMetricExporter());
 
+            services.AddLogging(logging =>
+            {
+                AzureMonitorOpenTelemetryOptions logExporterOptions = new();
+                if (configureAzureMonitorOpenTelemetry != null)
+                {
+                    configureAzureMonitorOpenTelemetry(logExporterOptions);
+                }
+
+                if (logExporterOptions.EnableLogs)
+                {
+                    logging.AddOpenTelemetry(builderOptions =>
+                    {
+                        builderOptions.IncludeFormattedMessage = true;
+                        builderOptions.ParseStateValues = true;
+                        builderOptions.IncludeScopes = true;
+                        builderOptions.AddAzureMonitorLogExporter(o => logExporterOptions.SetValueToExporterOptions(o));
```

Comment:
could you check if users will still be able to add a separate exporter like Console or OTLP?

Reply:
Adding below configuration sends data to both Azure Monitor and Console.
```
builder.Services.Configure<OpenTelemetryLoggerOptions>(options =>
{
    options.AddConsoleExporter();
});
```